sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Samples/SchemaRegistryClientSamples.HelloWorld.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Tests.Samples
+{
+    public class SchemaRegistryClientSamples : SamplesBase<SchemaRegistryClientTestEnvironment>
+    {
+        [Test]
+        public void GettingASecret()
+        {
+            var endpoint = TestEnvironment.SchemaRegistryUri;
+
+            #region Snippet:GetSecret
+            //var client = new SchemaRegistryClient(endpoint, new DefaultAzureCredential());
```

Comment:
nit:  Did you mean to leave a snippet with only comments here?

Reply:
Removed this file in favor of one that applies to the readme snippets.